README.md

```diff
@@ -37,7 +43,24 @@ The easiest way to try the web app is to run it through docker:
 - npm
 - HDF5 and NetCDF4 headers, if available on your platform
 
-On Debian, e.g., install these packages:
+
+### MacOS
+You can install these dependencies using brew. Don’t know what [brew](http://brew.sh/) is?
+
+Run:
+`
+brew install supervisor nginx postgresql npm hdf5
+`
+
+An alternative to installing these dependencies is to use pip. You can install each package by using the following:
```

Comment:
I think supervisor can be installed using pip.  Unfortunately, it does not run under Python 3, so if the user is in a Python 3 virtual env, this won't work.